armi/settings/fwSettings/globalSettings.py

```diff
@@ -406,18 +408,37 @@ def defineSettings() -> List[setting.Setting]:
         ),
         setting.Setting(
             CONF_BU_GROUPS,
-            default=[10, 20, 30, 100],
-            label="Burnup Groups",
+            default=[10, 20, 30],
+            label="Burnup XS Groups",
             description="The range of burnups where cross-sections will be the same "
             "for a given assembly type (units of %FIMA)",
             schema=vol.Schema(
                 [
                     vol.All(
-                        vol.Coerce(int), vol.Range(min=0, min_included=False, max=100)
+                        vol.Coerce(int),
+                        vol.Range(
+                            min=0,
+                            min_included=False,
+                        ),
                     )
                 ]
             ),
         ),
+        setting.Setting(
+            CONF_TEMP_GROUPS,
+            default=[],
+            label="Temperature XS Groups",
+            description="The fuel temperature boundaries where in between where cross-sections will be grouped together.",
```

Comment:
```suggestion
            description="The fuel temperature boundaries in between where cross-sections will be grouped together.",
```

Could also consider using language parallel to the description for burnup groups above.

Reply:
addressed